src/osmf/lib/HDSPlayer.as

```diff
@@ -95,6 +95,7 @@ package
       _mediaPlayerSprite.mediaPlayer.addEventListener(TimeEvent.COMPLETE, onPlayComplete);
       _mediaPlayerSprite.mediaPlayer.addEventListener(MediaErrorEvent.MEDIA_ERROR,onMediaError);
       _mediaPlayerSprite.mediaPlayer.addEventListener(BufferEvent.BUFFERING_CHANGE, bufferingChangeHandler);
+      CaptioningDocument .addEventListener(CaptioningDocument.CAPTION_READY,onCaptionready);
```

Comment:
is it okay that there is a space before "addEventListener"?  
Also, if possible, please add a space after the ','